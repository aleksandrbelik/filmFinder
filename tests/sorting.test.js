import React from 'react';
import Sorting from 'Common/sorting/sorting';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Sorting sort="sort" onSortRatingClick={() => {}} onSortReleaseDateClick={() => {}}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});