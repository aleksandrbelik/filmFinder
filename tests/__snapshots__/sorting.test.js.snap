// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sorting"
>
  <div>
    7 movies found
  </div>
  <div>
    <span>
      Sort by
    </span>
    <button
      className="sorting__button"
      onClick={[Function]}
      type="button"
    >
      release date
    </button>
    <button
      className="sorting__button"
      onClick={[Function]}
      type="button"
    >
      rating
    </button>
  </div>
</div>
`;
